src/form/FormikButton.tsx

```diff
@@ -85,6 +86,9 @@ export const FormikSubmitButton = ({
           color="primary"
           variant="contained"
           onClick={() => {
+            if (formik.values.advertiser.vertical === "Other") {
+              formik.values.advertiser.vertical = null;
+            }
```

Comment:
Can we make this _specific_ to the form that we need this in please?  Rather than changing the overall implementation of the form submit button with logic required only by a single form.

I'd expect to see this code in e.g. https://github.com/brave/ads-ui/blob/master/src/auth/hooks/mutations/useRegister.ts.